packages/common/src/storage/adapters/async-storage.ts

```diff
@@ -102,7 +107,7 @@ export const mountAsyncMultiStorage = <T = unknown>(
     dataFolder,
     keyExtractor,
     serializer = JSON.stringify,
-    deserializer = parseSafe as (item: string | null) => Nullable<T>,
+    deserializer = parseSafe as (item: string | null) => T | null,
```

Comment:
TS is dumb, it doesn't expand Generics, so Generics become opaque, and any null/undefined drop by asserts won't work with Generics, sad.